privaterelay/tests/utils_tests.py

```diff
@@ -311,6 +313,20 @@ def test_guess_country_from_accept_lang_short_primary_lang_fails(
     assert exc_info.value.accept_lang == accept_lang
 
 
+def test_accept_language_error_is_pickleable_mpp_3936() -> None:
+    bad_header = "en-gb;q=1.0000"
```

Comment:
question (non-blocking): I'm curious - what makes this an invalid accept-language header value?

Reply:
I copied this from earlier in the file:

https://github.com/mozilla/fx-private-relay/blob/51bb3d0bb2bc9a1d2ecd7eaa96e079a75a11ff0e/privaterelay/tests/utils_tests.py#L245-L246

which I copied from [Django's test suite](https://github.com/django/django/blob/14b46c1b848d846fad55c8634daeabd8787cf0d6/tests/i18n/tests.py#L1557-L1558) two years ago in PR #3682 - don't you remember?

You have to go to [RFC 2616](https://www.rfc-editor.org/rfc/rfc2616#section-3.9) to see why it is invalid - a "q value" should have no more that 3 digits after the decimal.